lighthouse-cli/index.js

```diff
@@ -113,9 +121,9 @@ if (cli.listAllAudits) {
 }
 
 if (cli.listTraceCategories) {
-  const categories = lighthouse.traceCategories;
+  const traceCategories = lighthouse.traceCategories;
 
-  process.stdout.write(JSON.stringify({traceCategories: categories}));
+  process.stdout.write(JSON.stringify({traceCategories}));
```

Comment:
I feel like since anything that will require this to be JSON will integrate with the core module, it might be nicer to output traceCategories and audits with log.info instead.